include/deal.II/dofs/dof_iterator_selector.h

```diff
@@ -56,10 +59,10 @@ namespace internal
      * @ref Iterators
      * module for more information.
      */
-    template <template <int, int> class DoFHandlerType, int spacedim, bool lda>
-    struct Iterators<DoFHandlerType<1, spacedim>, lda>
+    template <int spacedim, bool lda>
+    struct Iterators<1, spacedim, lda>
     {
-      using DoFHandler_type = DoFHandlerType<1, spacedim>;
+      using DoFHandler_type = DoFHandler<1, spacedim>;
```

Comment:
Would you mind making these each `DoFHandlerType`?

Reply:
On second thought - could we just deprecate these? They aren't used anywhere in the library AFAICT.

Reply:
Good catch!